configure

```diff
@@ -43,11 +45,12 @@ freshen_chef_repo() {
 	if [ ! -d chef_repo ]; then
 		git clone "$CHEF_REPO" -b "$CHEF_BRANCH" --recurse-submodules --remote-submodules chef_repo
 	else
-		pushd chef_repo
+		CURDIR="$(pwd)"
+		cd chef_repo
 		git fetch origin
```

Comment:
Alternatively, `git -C chef_repo <rest-of-the-command>` should work here to save some code lines, right?